### PR TITLE
Added Microsoft Fabric Synapse Data Warehouse documentation to the docs

### PR DESCRIPTION
**Pull request by prdpsvs**
Added Resource configs, setup and side bar links.
Updated supported-data platforms

### CODE REVIEW

**Comment by prdpsvs** [comment]
@dataders, please approve this PR today, if possible, for the blog post from Microsoft on the adapter release.

**Review by dataders — APPROVED** [review]
thanks @prdpsvs !

**Comment by prdpsvs** [comment]
Can someone merge this PR please?

**Comment by prdpsvs** [comment]
> thanks @prdpsvs !

Thank you!

File: website/docs/docs/core/connect-data-platform/fabric-setup.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
+---
+title: "Microsoft Fabric Synapse Data Warehouse setup"
+description: "Read this guide to learn about the Microsoft Fabric Synapse Data Warehouse setup in dbt."
+id: fabric-setup
+meta:
+  maintained_by: Microsoft
+  authors: 'Microsoft (https://github.com/Microsoft)'
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/fabric-setup.md]:
```suggestion
  authors: '[Microsoft] (https://github.com/Microsoft)'
```

File: website/docs/docs/core/connect-data-platform/fabric-setup.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+  pypi_package: 'dbt-fabric'
+  min_core_version: '1.4.0'
+  cloud_support: Not Supported
+  platform_name: 'Microsoft Fabric'
+  config_page: '/reference/resource-configs/fabric-configs'
+---
+
+:::info
+
+Below is a guide for use with "Synapse Data Warehouse" a new product within Microsoft Fabric (preview) ([more info](https://learn.microsoft.com/en-us/fabric/data-warehouse/data-warehousing#synapse-data-warehouse))
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/fabric-setup.md:18]:
```suggestion
The following is a guide to use [Synapse Data Warehouse](https://learn.microsoft.com/en-us/fabric/data-warehouse/data-warehousing#synapse-data-warehouse), a new product within Microsoft Fabric (preview).
```

File: website/docs/docs/core/connect-data-platform/fabric-setup.md [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+
+To learn how to set up dbt with Azure Synapse Dedicated Pools, see [Microsoft Azure Synapse DWH setup](azuresynapse-setup)
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/fabric-setup.md]:
```suggestion
To learn how to set up dbt with Azure Synapse Dedicated Pools, see [Microsoft Azure Synapse DWH setup](/docs/core/connect-data-platform/azuresynapse-setup) 
```

File: website/docs/docs/core/connect-data-platform/azuresynapse-setup.md [hunk 4]
```diff
@@ -15,6 +15,14 @@ meta:
   config_page: '/reference/resource-configs/azuresynapse-configs'
 ---
 
+:::info
+
+Below is a guide for use with "Azure Synapse Analytics Dedicated SQL Pools" (formerly SQL DW). See [What is dedicated SQL pool (formerly SQL DW) in Azure Synapse Analytics?](https://learn.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-overview-what-is) for more info.
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/azuresynapse-setup.md]:
```suggestion
The following is a guide to using Azure Synapse Analytics Dedicated SQL Pools, formerly SQL DW. For more info, refer to [What is dedicated SQL pool (formerly SQL DW) in Azure Synapse Analytics?](https://learn.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-overview-what-is) for more info.

Refer to [Microsoft Fabric Synapse Data Warehouse](/docs/core/connect-data-platform/fabric-setup) to set it up with dbt.
```

File: website/docs/docs/core/connect-data-platform/azuresynapse-setup.md [hunk 5]
```diff
@@ -15,6 +15,14 @@ meta:
...
+
+To learn how to set up dbt with Microsoft Fabric Synapse Data Warehouse, see [Microsoft Fabric Synapse Data Warehouse setup](fabric-setup).
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/azuresynapse-setup.md]:
```suggestion
```

File: website/docs/docs/core/connect-data-platform/fabric-setup.md [hunk 6]
```diff
@@ -0,0 +1,386 @@
+---
+title: "Microsoft Fabric Synapse Data Warehouse setup"
+description: "Read this guide to learn about the Microsoft Fabric Synapse Data Warehouse setup in dbt."
+id: fabric-setup
+meta:
+  maintained_by: Microsoft
+  authors: '[Microsoft] (https://github.com/Microsoft)'
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/fabric-setup.md]:
```suggestion
  authors: '[Microsoft](https://github.com/Microsoft)'
```

File: website/docs/docs/core/connect-data-platform/fabric-setup.md [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+| `query_timeout` | The number of seconds used to wait for a query before failing. Defaults to `0`, which means that the timeout is disabled or uses the default system settings. | Optional |  Not applicable  |
+| `schema_authorization` |  Optionally set this to the principal who should own the schemas created by dbt. [Read more about schema authorization](#schema-authorization). | Optional |  Not applicable  |
+| `encrypt` |  Whether to encrypt the connection to the server. Defaults to `true`. Read more about [connection encryption](#connection-encryption). | Optional |  Not applicable  |
+| `trust_cert` |   Whether to trust the server certificate. Defaults to `false`. Read more about [connection encryption](#connection-encryption).| Optional |  Not applicable  |
+
+### Connection encryption
+
+Microsoft made several changes in the release of ODBC Driver 18 that affects how connection encryption is configured.
+To accommodate these changes, starting in dbt-sqlserver 1.2.0 or newer the default vallues of `encrypt` and `trust_cert` have changed.
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/fabric-setup.md]:
```suggestion
To accommodate these changes, starting in dbt-sqlserver 1.2.0 or newer the default values of `encrypt` and `trust_cert` have changed.
```

File: website/docs/docs/core/connect-data-platform/fabric-setup.md [hunk 8]
```diff
@@ -0,0 +1,386 @@
...
+
+</File>
+
+</TabItem>
+
+</Tabs>
+
+### Automatic AAD principal provisioning for grants
+
+Please note that automatic AAD principal provisioning is not supported by Microsoft Fabric Synapse Data Warehouse at this time. Even though, in dbt 1.2 or newer you can use the [grants](https://docs.getdbt.com/reference/resource-configs/grants) config block to automatically grant/revoke permissions on your models to users or groups, the data warehouse does not support this feature at this time.
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/fabric-setup.md]:
```suggestion
Please note that automatic AAD principal provisioning is not supported by Microsoft Fabric Synapse Data Warehouse at this time. Even though in dbt 1.2 or newer you can use the [grants](https://docs.getdbt.com/reference/resource-configs/grants) config block to automatically grant/revoke permissions on your models to users or groups, the data warehouse does not support this feature at this time.
```

File: website/docs/docs/core/connect-data-platform/fabric-setup.md [hunk 9]
```diff
@@ -0,0 +1,386 @@
...
+| `login_timeout`        | The amount of seconds to wait until a response from the server is received when establishing a connection. `0` means that the timeout is disabled. |                    | `0`           |
+| `query_timeout`        | The amount of seconds to wait until a response from the server is received when executing a query. `0` means that the timeout is disabled.         |                    | `0`           |
+
+Valid values for `authentication`:
+
+* `ActiveDirectoryPassword`: Active Directory authentication using username and password
+* `ActiveDirectoryInteractive`: Active Directory authentication using a username and MFA prompts
+* `ActiveDirectoryIntegrated`: Active Directory authentication using the current user's credentials
+* `ServicePrincipal`: Azure Active Directory authentication using a service principal
+* `CLI`: Azure Active Directory authentication using the account you're logged in with in the Azure CLI
```
Comment (mirnawong1) [inline website/docs/docs/core/connect-data-platform/fabric-setup.md]:
```suggestion
* `CLI`: Azure Active Directory authentication using the account you're logged in within the Azure CLI
```

File: website/docs/reference/resource-configs/fabric-configs.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+Indexes are not supported by Microsoft Fabric Synapse Data Warehouse. Any Indexes provided as a configuration is ignored by the adapter.
+
+## Grants with auto provisioning
+
+Grants with auto provisioning is not supported by Microsoft Fabric Synapse Data Warehouse at this time.
+
+## Permissions
+
+The AAD identity (user or service principal) must be a Fabric Workspace admin to work on the database level at this time. Fine grain access control will be in-corporated in the future.
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/fabric-configs.md]:
```suggestion
The AAD identity (user or service principal) must be a Fabric Workspace admin to work on the database level at this time. Fine grain access control will be incorporated in the future.
```

File: website/docs/reference/resource-configs/fabric-configs.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
+---
+title: "Fabric configurations"
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/fabric-configs.md]: do we want to mirror the azure synapse layout so it looks like `Microsoft Fabric DWH configurations`
```suggestion
title: "Microsoft Fabric DWH configurations"
```

**Comment by prdpsvs** [comment]
@mirnawong1 , how long does it usually take to view the changes in PR? I do not see the changes made here - https://docs.getdbt.com/docs/supported-data-platforms (v1.4)

**Comment by mirnawong1** [comment]
Hey @prdpsvs great question and we're looking at this internally right now. will revert back and let you know!

**Comment by mirnawong1** [comment]
Hey @prdpsvs sorry about that - looks like we had an odd netlify blip. The site is live and reflects the changes now: https://docs.getdbt.com/docs/supported-data-platforms

**Comment by runleonarun** [comment]
Hey there @prdpsvs It looks like it's showing up now! A little netlify glitch. Let us know if you see any other weirdness! 🙇🏻

**Comment by prdpsvs** [comment]
Thanks, I see the changes too.

**Pull request merged**